independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/ProjectGenerator.java

```diff
@@ -25,6 +25,7 @@
     String IS_SPRING = "is_spring";
     String RESOURCE_PATH = "path";
     String JAVA_TARGET = "java_target";
+    String APP_CONFIG_PROPERTIES = "app_config_properties";
```

Comment:
could you use this `QuarkusCodestartData` to put the key instead

Reply:
(I need to do a cleaning pass to remove this legacy class and only use the `QuarkusCodestartData` or `CreateProject` not sure yet)

Reply:
I added the dependency so I could use QuarkusCodestartData

Reply:
Please note that these properties will be overridden by codestarts projects.